### PR TITLE
Updates to Brave Jupiter tests

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Motivation:

Brave updated to use jupiter, so tests won't run until we update here.

Modifications:

- Migrate brave module tests to jupiter

Result:

- Allows Brave updates to be tested here

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
@anuraaga @kojilin I set my PR to allow edits from maintainers.. I tried and failed at least to get IntelliJ to recognize the nested test. If you can help, much appreciated!

**Comment by minwoox** [comment]
It's not easy to migrate the parameterized test with the current design. 😓 
How about just creating two different subclasses?
```java
class Http1BraveClientIntegrationTest extends BraveClientIntegrationTest {...}
class Http2BraveClientIntegrationTest extends BraveClientIntegrationTest {...}
```

**Comment by codefromthecrypt** [comment]
@minwoox great idea! I've got it from here

**Comment by codefromthecrypt** [comment]
After this change, there are only a couple tests that use junit classic, so should be easy to remove that dep from the build!

File: site/src/pages/docs/advanced-zipkin.mdx [hunk 1]
```diff
@@ -116,10 +116,11 @@ For example, you can use it with [Kafka](https://kafka.apache.org/) producer:
...
-[Kafka](https://kafka.apache.org/), and report timing data using the `spanReporter` you specified.
+[Kafka](https://kafka.apache.org/), and report timing data using the `spanHandler` you specified.
 The following screenshot shows a trace of a request:
 
 ![](../../images/zipkin_1.png)
 
 ## See also
-- [Armeria Zipkin example](https://github.com/openzipkin-contrib/zipkin-armeria-example)
+- [Brave example Armeria](https://github.com/openzipkin/brave-example/tree/master/armeria)
+- [Brave example Armeria with Kafka](https://github.com/openzipkin/brave-example/tree/master/armeria-kafka)
```
Comment (codefromthecrypt) [inline site/src/pages/docs/advanced-zipkin.mdx:126]: ps I personally updated both of these, so they are current now.

**Review by jrhee17 — APPROVED** [review]
Looks straightforward enough! Thanks @codefromthecrypt 🙇 👍 🙇

File: dependencies.toml [hunk 2]
```diff
@@ -9,7 +9,7 @@ assertj = "3.24.2"
 awaitility = "4.2.0"
 blockhound = "1.0.8.RELEASE"
 bouncycastle = "1.70"
-brave = "5.16.0"
+brave = "6.0.0"
```
Comment (codefromthecrypt) [inline dependencies.toml:12]: ps this is the important part, the tests were impact of this.

**Review by ikhoon — APPROVED** [review]
Thanks for updating, @codefromthecrypt! 👍👍

**Pull request merged**